app/models/form_profiles/vha_10_7959c.rb

```diff
@@ -7,7 +7,7 @@ def metadata
     {
       version: 0,
       prefill: true,
-      returnUrl: '/your-information/description'
+      returnUrl: '/applicant-info'
```

Comment:
This URL was outdated; updating so the "Continue your application" link will work properly